app/views/comments/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="row static-width main-content-container">
+
+  <div class="col-md-7 col-md-offset-1">
+    <%= render partial: 'layouts/errors', locals: { element: @comment } %>
+    
+    <ul class="no-list-style no-padding">
+      <%= form_for :comment, url: question_comments_path(@comment.commentable) do |t| %>
```

Comment:
In this case, something like form_for [@comment.commentable, @comment] would remove the need for the URL override.